### ISSUE TITLE
Bot to add Map templates as batch to maps in map categories

### ISSUE THREAD

**Issue by susannaanas** [wikimaps-dev/maptemplate#3]

**Comment by Abbe98** [comment]
Upcoming Christmas holiday...



```
input var: category

foreach item:
  get wikitext
  if valid props:
    switch template(wikitext)
    push wikitext
  else:
    skip
```
(pywikibot)

**Comment by susannaanas** [comment]
We dropped this in favor of promoting to use VisualFileChange to change the metadata template names from "Information" to "Map". Changing from "Artwork" to "Map" should also work without problems.

**Comment by Abbe98** [comment]
When there is multiply Information templates the above solution breaks

**Comment by susannaanas** [comment]
Good point. Does your proposal take this into account already? This could be a node for the [GLAMpipe](https://github.com/artturimatias/GLAMpipe) project as well.
